loadJsonFile.py

```diff
@@ -0,0 +1,42 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# 파일 읽기 -> result.json 로드하기
+with open('result.json') as f:
+	read_data = f.read()
```

Comment:
indentation should be 4 spaces

Reply:
In sublime text, I used one tap! but looking vim, it's 8 space,,, I revise 4 space in vim.